index.html

```diff
@@ -4,286 +4,274 @@
       <section class="contact-me--form">
-        <div class="form-bg--image"></div>
-        <div>
-          <h2 class="contact-me">Contact me</h2>
-          <div class="form-intro">
-            <p>
-              If you have an application you are interested in developing, a
-              feature that you need built or a project that needs coding. I’d
-              love to help with it
-            </p>
-          </div>
-          <form action="https://formspree.io/f/moqrojrw" method="post">
-            <input
-              type="text"
-              name="user_name"
-              class="contact--field"
-              id="name"
-              maxlength="30"
-              required
-              placeholder="Alier"
-            /><br /><br />
-            <input
-              type="email"
-              pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"
-              name="user_email"
-              class="contact--field"
-              id="email"
-              placeholder="alierpm2022@gmail.com"
-              required
-            /><br /><br />
-            <textarea
-              name="user_message"
-              class="text-msg"
-              id="message"
-              cols="42"
-              rows="8"
-              placeholder="Write your message here"
-              maxlength="500"
-              required
-            ></textarea
-            ><br /><br />
-            <input type="submit" class="form-button" value="Get In Touch" />
-          </form>
-        </div>
+        <h2 class="contact-me">Contact me</h2>
+        <p class="form-intro-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d <br />
+          love to help with it
+        </p>
+        <form action="https://formspree.io/f/moqrojrw" method="post">
+          <input
+            type="text"
+            name="user_name"
+            class="contact--field"
+            id="name"
+            maxlength="30"
+            required
+            placeholder="Alier"
+          /><br /><br />
+          <input
+            type="email"
+            pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"
+            name="user_email"
+            class="contact--field"
+            id="email"
+            placeholder="alierpm2022@gmail.com"
+            required
+          /><br /><br />
+          <textarea
+            name="user_message"
+            class="text-msg"
+            id="message"
+            cols="42"
+            rows="8"
+            placeholder="Write your message here"
+            maxlength="500"
+            required
+          ></textarea
+          ><br /><br />
+          <button class="form-btn hoverme activeme" type="submit">
+            Get In Touch
+          </button>
```

Comment:
- [ ] Your for form is working correctly :100: but please add your different Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. check the screenshot  :point_down: 

![Screenshot from 2022-07-13 18-48-59](https://user-images.githubusercontent.com/6615394/178812200-d4e17a98-0c03-49f7-8b2f-36b048fc85c6.png)

Reply:
Thanks for this! So you mean, I will have to implement all these button functions for the same button or just at least one?